eth2deposit/cli/new_mnemonic.py

```diff
@@ -34,12 +35,13 @@ def new_mnemonic(ctx: click.Context, mnemonic_language: str, **kwargs: Any) -> N
     test_mnemonic = ''
     while mnemonic != test_mnemonic:
         click.clear()
+        # ToDo: Figure out why changing this line causes pytest to get stuck collecting.
```

Comment:
@hwwhww, any idea why using my translation function `load_text` on the following line causes pytest to hang `collecting`?

Reply:
Thank you so much. So simple, I just couldn't see it!

Reply:
@CarlBeek 
Because we check the output string in the cli tests, the checks failed after you modified the string. Fixed in https://github.com/ethereum/eth2.0-deposit-cli/pull/182/commits/4f101752389a564f6535875b02a4dcaf8d925ee6.

I tried to use `load_text` in tests directly, but it seems to have a problem finding the path. :'(